fast-downward.py

```diff
@@ -0,0 +1 @@
+forbiditerative/fast-downward.py
```

Comment:
Hate to be nitpicking, but can we make the one in forbiditerative the symbolic link, and the one in the root folder the real one? Or we can just have two copies, I don't expect this file to ever change.

Reply:
yeah sure either way